src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private transient Node<K, V>[] table;
+    private int size;
+    private double threshhold;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+        int size;
+        threshhold = table.length * LOAD_FACTOR;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size + 1 > threshhold) {
+            resize();
+        }
+        Node newNode = new Node<>(key, value, null);
```

Comment:
```suggestion
        Node<K, V> newNode = new Node<>(key, value, null);
```
don't forget about parameterization